### PR TITLE
Add the markup needed for students to submit their solutions

### PR DESCRIPTION
**Pull request by davidauza-engineer**
I'm adding the necessary markup for students to submit their solutions for this project.

Thank you.

### CODE REVIEW

**Comment by 105ron** [comment]
I don't really know why this project doesn't have submissions. Maybe there was a reason or someone just forgot to add it? 🤔

**Comment by happiguru** [comment]
I am submitting my solution to first part of the odin project (google homepage) [ https://happiguru.github.io/google_homepage/](url)

File: rails_programming/routes_views_controllers_assets/project_basic_rvc.md [hunk 1]
```diff
@@ -72,6 +72,17 @@ Hopefully this has been a useful adventure into the request cycle of a Rails app
 
 </div>
 
+### Student Solutions
+
+Send us your solution by editing this [file](https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/routes_views_controllers_assets/project_basic_rvc.md) so we can show others! Submit a link to the Github repo with your files in it here using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).  Please include your partner's github handle somewhere in the description if they would like attribution.
```
Comment (jordanliu) [inline rails_programming/routes_views_controllers_assets/project_basic_rvc.md:77]: Dead link https://github.com/TheOdinProject/curriculum/blob/master/rails_programming/routes_views_controllers_assets/project_basic_rvc.md

**Comment by Drew-Goddyn** [comment]
This file no longer exists and is in "old lessons" looks like RVC is broken out into separate files [here](https://github.com/TheOdinProject/curriculum/tree/master/rails_programming/rails_basics)

Closing... ping me if this is incorrect

**Pull request closed without merging**